src/Rules/Resources/HowToFix.resx

```diff
@@ -119,216 +119,277 @@
   </resheader>
   <data name="BoundingRectangleNotValidButOffScreen" xml:space="preserve">
     <value>The BoundingRectangle property is not valid, but the element is off-screen.</value>
+    <comment>The description of a problem with an element currently under inspection. The term "BoundingRectangle" should not be translated (i.e. it should be treated as a loanword).</comment>
```

Comment:
Same comments as other resx files:
1. Consider dropping loanword portion
2. If you keep it, consistent spacing before the open parenthesis